docker/build_bvtck.sh

```diff
@@ -33,15 +33,15 @@ mvn -version
 
 
 sed -i "s#^porting\.home=.*#porting.home=$WORKSPACE#g" "$WORKSPACE/build.xml"
-sed -i "s#^glassfish\.home=.*#glassfish.home=$WORKSPACE/glassfish5/glassfish#g" "$WORKSPACE/build.xml"
+sed -i "s#^glassfish\.home=.*#glassfish.home=$WORKSPACE/glassfish6/glassfish#g" "$WORKSPACE/build.xml"
```

Comment:
Not immediate, we could create a release tag for jakartaee8 release from prior to such changes.

Reply:
done, have created tag for jakartaee8 release.